### PR TITLE
Remove unused codes

### PR DESCRIPTION
**Pull request by wxyucs**
Signed-off-by: Xiangyu Wang <xiangyu.wang@zilliz.com>

/kind improvement
/cc @congqixia

### CODE REVIEW

File: cmd/roles/roles.go [hunk 1]
```diff
@@ -436,14 +416,6 @@ func (mr *MilvusRoles) Run(localMsg bool, alias string) {
 		}
 	}
 
-	var mss *components.MsgStream
-	if mr.EnableMsgStreamCoord {
```
Comment (congqixia) [inline cmd/roles/roles.go:440]: `mr.EnableMsgStreamCoord` shall be deleted as well.

File: cmd/roles/roles.go [hunk 2]
```diff
@@ -322,26 +322,6 @@ func (mr *MilvusRoles) runIndexNode(ctx context.Context, localMsg bool, alias st
...
-		mss, err = components.NewMsgStreamCoord(ctx)
-		if err != nil {
-			panic(err)
-		}
-		wg.Done()
-		_ = mss.Run()
-	}()
-	wg.Wait()
-
-	metrics.RegisterMsgStreamCoord()
```
Comment (congqixia) [inline cmd/roles/roles.go:341]: Metrics of MsgStreamCoord shall be purged too

**Comment by scsven** [comment]
Hi @congqixia, I solved all the comments, please review this changes again.

**Pull request merged**